### PR TITLE
Remove oracle

### PR DESCRIPTION
**Pull request by mcalmer**
## What does this PR change?

Our schema package is still based on oracle syntax and migrate it to postgresql using sed and other tricks.
As postgresql is currently the only supported DB in uyuni we should drop everything around oracle now.

This PR start with this:

- migrate all scripts to postgresql syntax
- remove oracle specific files
- remove validators to check sha1 checksum of oracle equivalent file.

The schema upgrade still work with .postgresql suffix as the script requires this.

## GUI diff

No difference.

- [x] **DONE**

## Documentation
- No documentation needed: **internal**

- [x] **DONE**

## Test coverage
- No tests: **already covered**

- [x] **DONE**

## Links

Fixes https://github.com/SUSE/spacewalk/issues/8883

- [x] **DONE**

## Changelogs

If you don't need a changelog check, please mark this checkbox:

- [x] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_lint_checkstyle"		 
- [ ] Re-run test "java_pgsql_tests"		 
- [ ] Re-run test "ruby_rubocop"
- [ ] Re-run test "schema_migration_test_oracle"
- [ ] Re-run test "schema_migration_test_pgsql"		 
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"		 
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

**Comment by moio** [comment]
For now I am adding a commit about something I had to remove anyway, please wait next week for a proper review and possibly more removals!

Thanks for this quarter-million LOC drop!

**Comment by mcalmer** [comment]
@moio JFYI: cucumber tests for this PR plus https://github.com/uyuni-project/uyuni/pull/1897 is running in our test environment.

**Comment by moio** [comment]
There's likely more code to be removed. Some easy references that I spotted during review:

* the whole file: `backend/server/importlib/backendOracle.py`
* occurences of `CFG.DB_BACKEND`
* occurences of `OracleBackend`
* occurences of `oracleSQLException`

File: schema/spacewalk/upgrade/spacewalk-schema-1.3-to-spacewalk-schema-1.4/022-rhn_entitlements.sql [hunk 1]
```diff
@@ -0,0 +1,1750 @@
...
+    -- PROCEDURE: activate_system_entitlement
+    --
+    -- Sets the values in rhnServerGroup for a given rhnServerGroupType.
+    --
+    -- Calls: set_group_count to update, prune, or create the group.
+    -- Called by: the code that activates a satellite cert.
+    --
+    -- Raises not_enough_entitlements_in_base_org if all entitlements
+    -- in the org are used so the free entitlements would not cover
+    -- the difference when descreasing the NUMERIC of entitlements.
```
[the remark this answers is not in the document]
Comment (mcalmer) [reply schema/spacewalk/upgrade/spacewalk-schema-1.3-to-spacewalk-schema-1.4/022-rhn_entitlements.sql]: I fixed the one I could find.

**Comment by mcalmer** [comment]
> There's likely more code to be removed. Some easy references that I spotted during review:
> 
> * the whole file: `backend/server/importlib/backendOracle.py`
> * occurences of `CFG.DB_BACKEND`
> * occurences of `OracleBackend`
> * occurences of `oracleSQLException`

I would like to do this in a separate PR as this one is already too big. And my feeling is that when starting to remove them, we find things deeper and deeper which can also be removed.

As this PR already passed the testsuite I would like to not touch it too much anymore.

**Comment by mcalmer** [comment]
The actions fail because this pr has too many files.
The schema migration test fail because the idepodent test try to test all touched files and this PR
change files before we started to write them in this way.

Force merging

**Comment by mcalmer** [comment]
Just for reference: created a new issue for leftovers: https://github.com/SUSE/spacewalk/issues/10829

**Pull request merged**